src/ExpenseTrackerGrupo2/DataAccess/Layer/ExpenseRepository.cs

```diff
@@ -0,0 +1,19 @@
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.Data;
+using ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Entities;
+
+namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Layer;
+
+public class ExpenseRepository : BaseRepository<Expense>, IExpenseRepository
+{
+    public ExpenseRepository(IDbConnectionFactory dbConnectionFactory) : base(dbConnectionFactory) {}
+
+    public Task<IList<Expense>> GetExpenseByCategory(string category)
+    {
+        throw new NotImplementedException();
+    }
+
+    public Task<int> GetExpenseByDate(DateTime startDate, DateTime endDate)
+    {
+        throw new NotImplementedException();
```

Comment:
There is no implementation of GetExpenseByDate

Reply:
Done